earthpy/tests/test_plot.py

```diff
@@ -105,11 +100,11 @@ def test_single_band_2dims(one_band_3dims):
 """ Colorbar Tests """
 
 
-def test_colorbar_height(one_band_2dims):
+def test_colorbar_height(basic_image):
```

Comment:
Good question @lwasser! `basic_image` is a fixture that we borrowed from rasterio, and it is defined in conftest.py: https://github.com/earthlab/earthpy/blob/893c278717f88d209a8f4c8c6d3d32eed1f0d331/earthpy/tests/conftest.py#L127

The fixture that was called `one_band_2dims` was redundant, as `basic_image` is available to all test modules. The name `basic_image` is what rasterio uses - we could change it if we want but it might make it harder to borrow some of the existing test logic that rasterio has developed. Maybe we can discuss in our meeting - this fixture is pretty important for the rasterio tests, and existing earthpy tests for cropping: https://github.com/earthlab/earthpy/blob/3ab23a3748b0a58a14ceeb99d1133179a5ae90b7/earthpy/tests/test_spatial.py#L171-L252